packages/web-components/src/components/ic-classification-banner/ic-classification-banner.tsx

```diff
@@ -37,7 +37,19 @@ export class ClassificationBanner {
   @Prop() upTo?: boolean = false;
 
   render() {
-    const { classification, inline, country, upTo, additionalSelectors } = this;
+    const { inline, upTo } = this;
+
+    // In case of unrecognized props, fallback to default
+    let { country, additionalSelectors, classification } = this;
+    if (!country || country === undefined) country = "";
+    if (!additionalSelectors || additionalSelectors === undefined)
+      additionalSelectors = "";
```

Comment:
FYI: This was "uk" as default before, but in the unit tests, is should render no country if none is supplied.